README.md

```diff
@@ -53,12 +53,18 @@ $ ./scripts/server
 ### Running directly
 
 * `cd taui`
-* Install packages: `yarn add`
+* Install packages: `yarn install`
```

Comment:
So I tried both of these; Docker works but takes a _long_ time plus it does make my Mac sound like a jet engine (Activity Monitor is reporting Docker as using 300%+ CPU which I looked into because 😲 . Looks like that means I'm using 3 of my 4 cores). However, it's better than running on host because that didn't work at all--it failed while trying to build node-sass. While helping me spin up, Klaas mentioned that the lack of versioning on node and yarn could cause potential consistency problems with trying to run on host and that Docker was the lesser of the two evils. So I've been doing all of my work while on Docker and didn't try to pursue the build fail on host.

Reply:
Ok, I asked Rachele if she was able to run it on host and she is so something is happening with my computer. I'm going to try to look at it with Klaas during pairing tomorrow morning to figure out what it's getting caught up on and see if I can get running on host to work for me.